lib/view.js

```diff
@@ -0,0 +1,73 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-connector-cloudant
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+var URL = require('url');
+var assert = require('assert');
+var util = require('util');
+var _ = require('lodash');
+
+module.exports = mixinView;
+
+function mixinView(Cloudant) {
+  var debug = require('debug')('loopback:connector:cloudant:view');
+
+  /**
+   * Gets data at `/{db}/_design/{ddocName}/views/{viewName}`
+   *
+   * Example:
+   * User has a view called `getModel` in design document /{db}/_design/model,
+   * to query the view, user can call function:
+   * ```
+   * ds.viewDocs(model, getModel, {key: 'purchase'}, cb);
+   * ```
+   *
+   * @param {String} ddocName The design doc name
```

Comment:
@rmg I am confused why a user would think of using part of the url as a design doc name? By reading couchdb/cloudant document, or those nodejs drivers' README.md, I don't see anywhere implies that the design document name would contain the those prefix....